integration-tests/server/test/index.test.js

```diff
@@ -2,9 +2,11 @@ import { Client } from '@elastic/elasticsearch';
 import express from 'express';
 import { print } from 'graphql';
 import gql from 'graphql-tag';
+import {} from 'mocha'; // Removes error ts2304 from "before" and "after" testing hooks
 
-import Arranger, { adminGraphql } from '../../../modules/server/dist';
-import ajax from '../../../modules/server/dist/utils/ajax';
+// import Arranger, { adminGraphql } from '../../../modules/server/dist';
```

Comment:
keeping the admin bits as comments, for when we're ready to revamp that service.